pkg/search/result.go

```diff
@@ -104,6 +169,63 @@ func (repo Repository) ExportData(fields []string) map[string]interface{} {
 				"id":    repo.Owner.ID,
 				"login": repo.Owner.Login,
 				"type":  repo.Owner.Type,
+				"url":   repo.Owner.URL,
+			}
+		default:
+			sf := fieldByName(v, f)
+			data[f] = sf.Interface()
+		}
+	}
+	return data
+}
+
+func (issue Issue) IsPullRequest() bool {
+	return issue.PullRequestLinks.URL != ""
+}
+
+func (issue Issue) ExportData(fields []string) map[string]interface{} {
+	v := reflect.ValueOf(issue)
+	data := map[string]interface{}{}
+	for _, f := range fields {
+		switch f {
+		case "assignees":
+			assignees := make([]interface{}, 0, len(issue.Assignees))
+			for _, assignee := range issue.Assignees {
+				assignees = append(assignees, map[string]interface{}{
+					"id":    assignee.ID,
+					"login": assignee.Login,
+					"type":  assignee.Type,
+					"url":   assignee.URL,
```

Comment:
I don't think it's useful to include the REST API URL for each user record. I would vote to omit it